src/app_io.py

```diff
@@ -54,6 +59,7 @@ def pop_output(self) -> str | None:
         """This allows test script to get outputs from the queue"""
 
         if len(self._outputs) == 0:
-            raise RuntimeWarning("Your test asked for output, but there are none left!")
+            raise RuntimeWarning(
+                "Your test asked for output, but there are none left!")
```

Comment:
I think you somehow have the formatter set up differently

Reply:
yes, I was actually wondering about it, were we supposed to all independently configure the IDE, or have I previously set up some "exceptions"? oh, well, `water under the bridge` at this point :)